### ISSUE TITLE
Spotify oEmbed stopped working

### ISSUE THREAD

**Issue by martinsnajdr** [spotify/web-api#1479]
I know this is web-api support repo, but I'm using WP oEmbed functionality and spotify integration suddenly stopped working.

Just try the spotify example links here, to test it:
https://oembed.com

You will get "Sorry, couldn't find that" page.

Any information why it stopped working?

**Comment by alexplumb** [comment]
`GET https://open.spotify.com/embed/oembed?url=https://open.spotify.com/artist/0jYpQ8seujbnWxcrfxT7Cw`

returns a status code of 404

**Comment by kopiro** [comment]
Hey, the problem is the trailing slash at the end of the path (`oembed/`).
This works instead: https://open.spotify.com/oembed?url=https://open.spotify.com/track/2qToAcex0ruZfbEbAy9OhW

We have rolling out a fix to handle both paths.

**Comment by martinsnajdr** [comment]
Thank you for a reply. Well, the URL you posted is completely without the `embed/` part. 

Ok, so it should work as before without any tweaks?

**Comment by alexplumb** [comment]
It looks like it now works without the `embed/` url part.

**Comment by serchavalos** [comment]
Right; hitting `https://open.spotify.com/oembed?url=https://open.spotify.com/artist/0jYpQ8seujbnWxcrfxT7Cw` will give you the right response

Like @kopiro noted, you should see the previous link working as well:
```
GET https://embed.spotify.com/oembed/?url=https://open.spotify.com/track/2qToAcex0ruZfbEbAy9OhW
GET https://embed.spotify.com/oembed/?url=spotify%3Aartist:7ae4vgLLhir2MCjyhgbGOQ
```

**Comment by kopiro** [comment]
Con you confirm issue has been solved?

**Comment by PatrickGammersbach** [comment]
I try to embed via ajax-Request. In the console I get a CORS-Error. How do I embed correctly via ajax?
